### PR TITLE
fix: document metadata

### PR DESCRIPTION
**Pull request by RogerHYang**
1. reverts #1355
2. adds unit [test](https://github.com/Arize-ai/phoenix/pull/1383/files#diff-58b05421cdf6b39e021e2910a6b31f0172c2e71d14a361e02246eb622356f332R50-R51) (main is currently failing on this test)
3. updates ruff to 0.0.290
4. updates fixture to `llama_index_rag_v6.jsonl`

### CODE REVIEW

File: src/phoenix/trace/span_json_decoder.py [hunk 1]
```diff
@@ -39,14 +29,14 @@ def json_to_attributes(obj: Optional[Dict[str, Any]]) -> Dict[str, Any]:
...
-    if documents := obj.get(RETRIEVAL_DOCUMENTS):
-        obj[RETRIEVAL_DOCUMENTS] = [json_to_document(document) for document in documents]
     return obj
 
 
 def json_to_span(data: Dict[str, Any]) -> Any:
     """
     A hook for json.loads to convert a dict to a Span object.
+
+    NB: this function is mainly used for testing purposes. Consider swapping this out for pydantic.
```
Comment (mikeldking) [inline src/phoenix/trace/span_json_decoder.py:39]: did you re-add this? I removed it intentionally.

File: tests/trace/test_span_json_decoder.py [hunk 2]
```diff
@@ -1,59 +0,0 @@
-from phoenix.trace.span_json_decoder import json_to_span
-
-
-def test_span_json_decoder_document_retrieval():
```
Comment (mikeldking) [inline tests/trace/test_span_json_decoder.py:4]: I'd prefer to preserve the test - just fix the content - this code path does get executed in some of the notebooks.

**Review by mikeldking — APPROVED** [review]

File: src/phoenix/trace/span_json_decoder.py (hunk 1 above)
Comment (RogerHYang) [reply src/phoenix/trace/span_json_decoder.py:39]: it's just from the revert. i can remove

File: tests/trace/test_span_json_decoder.py (hunk 2 above)
Comment (RogerHYang) [reply tests/trace/test_span_json_decoder.py:4]: ok

**Pull request merged**